servers/parser-graphql-wrapper/schema-admin.graphql

```diff
@@ -0,0 +1,556 @@
+extend schema
```

Comment:
I had to add a seperate schema for admin because federation gets mad since our admin api does not implement some of the shared objects